tests/infographics/test_api.py

```diff
@@ -0,0 +1,427 @@
+                                ("type", "richtext"),
+                                ("required", True),
+                                ("read_only", False),
+                                ("label", "Copy x"),
+                                ("field_size", "long")
+                            ])
+                        ), (
+                            "copy_y", OrderedDict([
+                                ("type", "richtext"),
+                                ("required", True),
+                                ("read_only", False),
+                                ("label", "Copy y"),
+                                ("field_size", "long")
+                            ])
+                        )
+                    ]))
+                ]),
+                "key_x": OrderedDict([
+                    (
+                        "title", OrderedDict([
+                            ("type", "richtext"),
+                            ("required", True),
+                            ("read_only", False),
+                            ("label", "Title"),
+                            ("field_size", "short")
+                        ])
+                    ), (
+                        "color", OrderedDict([
+                            ("type", "string"),
+                            ("required", True),
+                            ("read_only", False),
+                            ("label", "Color")
+                        ]),
+                    ), (
+                        "initial", OrderedDict([
+                            ("type", "string"),
+                            ("required", True),
+                            ("read_only", False),
+                            ("label", "Initial")
+                        ])
+                    )
+                ]),
+                "key_y": OrderedDict([
+                    (
+                        "title", OrderedDict([
+                            ("type", "richtext"),
+                            ("required", True),
+                            ("read_only", False),
+                            ("label", "Title"),
+                            ("field_size", "short")
+                        ])
+                    ), (
+                        "color", OrderedDict([
+                            ("type", "string"),
+                            ("required", True),
+                            ("read_only", False),
+                            ("label", "Color")
+                        ]),
+                    ), (
+                        "initial", OrderedDict([
+                            ("type", "string"),
+                            ("required", True),
+                            ("read_only", False),
+                            ("label", "Initial")
+                        ])
+                    )
+                ]),
+            }
+        })
+
+    def get_detail_endpoint(self, pk):
+        return reverse("content-detail", kwargs={"pk": pk})
+
+    @property
```

Comment:
Didn't want to rewrite the same values over and over, the alternative is write them in the setUp, which yields the same behavior per test.